docs/core.md

```diff
@@ -313,6 +313,19 @@ app({
   actions: {
     addOne: state => state + 1
   },
-  plugins: [Logger]
+  mixins: [Logger]
 })
 ```
+Mixins can also extend other mixins:
```

Comment:
@jamen I know it's hard, very hard, to come up with good examples, but since you're introducing this feature, can you try to come up with a more practical example? 🙏